its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -909,13 +909,8 @@ void testAzureFunctions_WithWrongBaseDirectory_AnalysisSucceeds() throws IOExcep
     assertThat(buildResult.isSuccess()).isTrue();
     // ToDo this will be fixed by https://github.com/SonarSource/sonar-scanner-msbuild/issues/1309
     // Expected: projectDir should be the base directory
-    if (isRunningUnderAzureDevOps()) {
-      // this might fail if Azure changes the drive
-      assertThat(buildResult.getLogs()).contains("Using longest common projects path as a base directory: 'C:\\'");
-    } else {
-      var temporaryFolderRoot = basePath.getParent().toFile().getCanonicalFile().toString();
-      assertThat(buildResult.getLogs()).contains("Using longest common projects path as a base directory: '" + temporaryFolderRoot + "'");
-    }
+     var temporaryFolderRoot = basePath.getParent().toFile().getCanonicalFile().toString();
+     assertThat(buildResult.getLogs()).contains(" '" + temporaryFolderRoot);
```

Comment:
Do those changes have an impact on the mend alerts?
If not, can you explain to me why you could change it?

Does that mean that #1309 is fixed somehow?

Reply:
Yes! 
Now the autogenerated code is in the debug folder and not in the Temp.